internal/config/main_test.go

```diff
@@ -0,0 +1,159 @@
+package config
+
+import (
+	"flag"
+	"os"
+	"testing"
+
+	"github.com/bluecmd/fortigate_exporter/internal/utils/test"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	validAuthFile      = test.GetFixturePathPanic("validAuthFile.yaml")
+	otherValidAuthFile = test.GetFixturePathPanic("otherValidAuthFile.yaml")
+	invalidAuthFile    = test.GetFixturePathPanic("invalidAuthFile.yaml")
+	certFile           = test.GetRelativeFixturePathPanic("cert.pem")
+)
+
+func TestInit(t *testing.T) {
```

Comment:
This is 3 functions smashed into one it seems, can we split it to `TestInit_AuthFile` / `TestInit_CACerts` etc?

Reply:
Sadly not because this has to be executed serially, as I modify the global args and flag objects

Reply:
Can we figure out a way that breaks out of the mess of doing global state? It really would improve the current exporter code a lot.